MVC/view/CLI.py

```diff
@@ -76,8 +76,80 @@ def showHoneyComb(self):
                     %s       %s
                         %s      
             ''' % (self.displayLetters[0], self.displayLetters[1],self.displayLetters[2], self.controller.controllerGetReqLetter(), self.displayLetters[3],self.displayLetters[4],self.displayLetters[5]))
+    def newPuzzle(self):
+        if(self.controller.controllerGetPuzzleState() == 1):
+            wantSave = input("Hey do you want to save the game? (yes/no): ")
+            if (wantSave.lower() == "yes"):
+                inputFile = input("Please choose a name for the file: ")
+                print("Saving your game!")
+                self.controller.controllerSaveGame(inputFile)
+            else:
+                print("Ok, lets generate a new puzzle! ")
+        self.controller.controllerNewGame()
+        isAuto = input("Do you want it to be automatically generated? (yes/no): ")
+        while isAuto.lower() != "yes" and isAuto.lower() != "no":
+            isAuto = input("Do you want it to be automatically generated? (yes/no): ")
+    
+        if (isAuto.lower() == "yes"):
+            self.controller.controllerRunAutoGame()
```

Comment:
These 5 lines could be refactored since they appear again below (just use a parameter to check for AutoGame or BaseGame